docs/source/markdown/podman-commit.1.md

```diff
@@ -86,6 +86,11 @@ $ podman commit -q --pause=false containerID image-committed
 e3ce4d93051ceea088d1c242624d659be32cf1667ef62f1d16d6b60193e2c7a8
 ```
 
+```
+$ podman commit -q --change LABEL=io.containers.capabilities=setuid,setgid epic_nobel privimage
```

Comment:
You don't have the verbiage from build in here, should you?

Reply:
I added it.

Reply:
This is badly formatted; should be `--change 'LABEL io.containers.capabilities=setuid,setgid'`

Reply:
Their might be a bug in the parser, but my syntax works, and yours does not.  This is also documented two lines higher.
```
$ podman commit --change "LABEL foo=bar" myweb dan1
Error: invalid syntax for --change: LABEL foo=bar

$ podman commit --change "LABEL=foo=bar" myweb dan1
Getting image source signatures
Copying blob 195be5f8be1d skipped: already exists  
Copying blob 5f70bf18a086 [--------------------------------------] 0.0b / 0.0b
Copying config e87256c3fc done  
Writing manifest to image destination
Storing signatures
e87256c3fcda3881112f19dbfda9a733bee0472354269dc62c2c5c7e81c69e1f
```


oo=bar" myweb dan1
Error: invalid syntax for --change: LABEL foo=bar